x-pack/plugins/ml/server/models/data_frame_analytics/validation.ts

```diff
@@ -306,6 +346,33 @@ async function getValidationCheckMessages(
 
     if (analyzedFields.length && analyzedFields.length > INCLUDED_FIELDS_THRESHOLD) {
       analysisFieldsNumHigh = true;
+    } else {
+      if (analysisType === ANALYSIS_CONFIG_TYPE.OUTLIER_DETECTION && analyzedFields.length < 1) {
+        lowFieldCountWarningMessage.text = i18n.translate(
+          'xpack.ml.models.dfaValidation.messages.lowFieldCountOutlierWarningText',
+          {
+            defaultMessage:
+              'Outlier detection requires at least one field to be included in the analysis.',
+          }
+        );
+        messages.push(lowFieldCountWarningMessage);
+      } else if (
+        analysisType !== ANALYSIS_CONFIG_TYPE.OUTLIER_DETECTION &&
+        analyzedFields.length < 2
+      ) {
+        lowFieldCountWarningMessage.text = i18n.translate(
+          'xpack.ml.models.dfaValidation.messages.lowFieldCountWarningText',
+          {
+            defaultMessage:
+              '{analysisType} requires at least two fields to be included in the analysis.',
```

Comment:
```suggestion
              '{analysisType} requires that at least two fields are included in the analysis.',
```

Reply:
Updated in 97a862cfc489536c59a7b556413f277f8126b5a8